### ISSUE TITLE
setVisible is missing from SplitviewPanelApiImpl

### ISSUE THREAD

**Issue by garyvh2** [mathuo/dockview#534]
**Describe the bug**
Originally used `setVisible` to control the visibility of the split view panels

**To Reproduce**
Our use case does the  following

Finds the current visible panel by this code

```
const currentVisible = api.current.panels.find(
  p => p.api.isVisible,
)
```

Then proceeds to `setVisible` based on certain logic

```
currentVisible.api.setVisible(false)
```

Couldn't find any release notes on this being deprecated, so not sure if it was a mistake or which function should be used on it's place.

**Expected behavior**
Panel API `setVisible` is available

**Screenshots**
![image](https://github.com/mathuo/dockview/assets/21265830/d2797a51-9061-4ca0-bca1-0dc681a45066)

**Desktop (please complete the following information):**
 - Browser all
 - Version latest 1.10.0

**Comment by garyvh2** [comment]
Used `isHidden` instead, let me know if that's the expected replacement.

**Comment by michaelsharpe** [comment]
Having the same issue, setVisible disappeared which we used to show and hide panes form hotkeys.  isHidden is read only so it doesn't seem to work.

**Comment by mathuo** [comment]
version `1.10.0` did make some efforts to cleanup the events dockview fires (isVisible, isActive etc) to reduce duplicates and fix a couple of bugs where events didn't fire. In that process it seems I have mistakenly removed `setVisible` from the public api and it wasn't caught by any tests.

I will reinstate `setVisible` along with ensuring  `isVisible` and `onDidVisibilityChange` work as expected through additional unit tests. `onHidden` should not be on the public api and this will be removed and will again be validated through additional unit tests.

I will aim to release a version tomorrow, sorry about the bug.

**Comment by mathuo** [comment]
Released in 1.10.1
